docs/3.0-migration-guide.md

```diff
@@ -141,7 +141,7 @@ recommended), or users who used it through the entropy module but had it as the
 only source of entropy. If you're in that case, please declare OS or hardware
 RNG interfaces with `mbedtls_entropy_add_source()` and/or use an entropy seed
 file created securely during device provisioning. See
-<https://tls.mbed.org/kb/how-to/add-entropy-sources-to-entropy-pool> for more
+<https://mbed-tls.readthedocs.io/en/latest/kb/how-to/add-entropy-sources-to-entropy-pool> for more
```

Comment:
Should this link have a trailing `/` like the others?

Reply:
The links work correctly either way. Agreed, it would be nicer if I'd been consistent but not sure it's worth fixing?

Reply:
> it would be nicer if I'd been consistent but not sure it's worth fixing?

You're the PR author - that decision is yours. It's not a blocker, just inconsistent, so I've pointed it out